autoload/nerdfont/path/pattern.vim

```diff
@@ -1,21 +1,11 @@
 scriptencoding utf-8
 
 let g:nerdfont#path#pattern#customs = get(g:, 'nerdfont#path#pattern#customs', {})
-let g:nerdfont#path#pattern#defaults = {
-      \ '.*angular.*\.js$'      : '',
-      \ '.*backbone.*\.js$'     : '',
-      \ '.*jquery.*\.js$'       : '',
-      \ '.*materialize.*\.css$' : '',
-      \ '.*materialize.*\.js$'  : '',
-      \ '.*mootools.*\.js$'     : '',
-      \ '.*require.*\.js$'      : '',
-      \ '.*vimrc.*'             : '',
-      \ 'Vagrantfile$'          : '',
-      \}
+let g:nerdfont#path#pattern#defaults = nerdfont#get_json('pattern')
 
 function! nerdfont#path#pattern#find(path) abort
   for [k, v] in s:m
-    if a:path =~# k
+    if a:path =~# '\v' . k
```

Comment:
Please do not use "very magic". I prefer to force developers to use Vim's regex.

Reply:
OK, I'll fix it. I add `\v` just due to I am afraid someone change `&magic` in their vimrc. Should we change `\v` to `\m`?

Reply:
That's very good idea.